ee/migrations/0005_copy_over_non_ee_definitions_tables.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.1.8 on 2021-05-29 16:13
+
+from django.db import migrations
+
+
+def forward(apps, schema_editor):
```

Comment:
Not sure about this, I think this gets quite confusing. I would just do the OneToOne directly, and instead of relying on all objects to be in sync you use the `posthog` app's models as base and join this object when available.